x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlActionTestCase.java

```diff
@@ -144,6 +144,9 @@ protected EqlSearchResponse runQuery(String index, String query, boolean isCaseS
         EqlSearchRequest request = new EqlSearchRequest(testIndexName, query);
         request.isCaseSensitive(isCaseSensitive);
         request.tiebreakerField("event.sequence");
+        // some queries return more than 10 results
+        request.size(50);
+        request.fetchSize(2);
```

Comment:
Does it make sense to make it a random between let's say 1 and 10?

Reply:
It does - it has to be 2 since we're talking about a window (1 would work but the algo would have to change - that requires an extra call that I'd rather avoid).
I've pushed it with 2 just to get the build running enough times with this minimum size - I'll follow-up in the until PR with random.